### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t1726

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/cf5ab5cfa10d512/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/cf5ab5cfa10d512/output.txt

Total script time: 0.65 mins

Server is running on http://pr818.butyr.org/templates/basic/
Run the Test Suite: http://pr818.butyr.org/test/

File: cornfield/views/embed.jade [hunk 1]
```diff
@@ -66,10 +66,10 @@ html(lang='en')
                 .share-size
                   label Size
                   select#share-size.share-size-picker(selectedindex='2')
-                    option(value='560x340') 560x340
-                    option(value='640x385') 640x385
-                    option(value='853x505') 853x505
-                    option(value='1280x745') 1280x745
+                    option(value='560x58') 560x358
```
Comment (mjschranz) [inline cornfield/views/embed.jade]: Should be 560x358 right?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/72ce8abbe39e223/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/72ce8abbe39e223/output.txt

Total script time: 0.69 mins

Server is running on http://pr818.butyr.org/templates/basic/
Run the Test Suite: http://pr818.butyr.org/test/

File: css/controls.css [hunk 2]
```diff
@@ -160,23 +139,129 @@
   left: -5px;
 }
 
+/* Time */
+
+.time {
+  position: absolute;
+  left: 43px;
+  height: 27px;
+  background: url(../resources/controls/controls_time_arrowL.gif) no-repeat left top, url(../resources/controls/controls_time_arrowR.gif) right top no-repeat;
```
Comment (mjschranz) [inline css/controls.css:148]: Do these urls not need to be in quotes?

File: css/controls.css [hunk 3]
```diff
@@ -187,3 +272,83 @@
 .controls-unmuted {
   background-position: -25px -21px;
 }
+
+.controls-right div {
+  background: #cecfcf url(../resources/controls/controls_icons.png) no-repeat;
```
Comment (mjschranz) [inline css/controls.css:275]: same as above for urls.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/3d5dc3f406752d0/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/3d5dc3f406752d0/output.txt

Total script time: 1.21 mins

File: css/controls.css [hunk 4]
```diff
@@ -160,23 +139,129 @@
...
+#volume #volume-one {
+
+}
+
 /* Buttons */
 
+#controls-play {
+  width: 28px;
+  height: 28px;
+  background: #1eb771 url(../resources/controls/controls_icon_playPause.png) no-repeat;
```
Comment (mjschranz) [inline css/controls.css:243]: Same as above url comment

File: css/controls.css [hunk 5]
```diff
@@ -72,36 +41,44 @@
 }
 
 .controls-middle {
-  padding-top: 5px;
-  height: 17px;
+  margin-top: 0px;
```
Comment (mjschranz) [inline css/controls.css]: remove the px

File: css/controls.css [hunk 6]
```diff
@@ -160,23 +139,129 @@
...
+}
+
 /* Buttons */
 
+#controls-play {
+  width: 28px;
+  height: 28px;
+  background: #1eb771 url(../resources/controls/controls_icon_playPause.png) no-repeat;
+  padding: 0;
+  -webkit-border-radius: 3px;
```
Comment (mjschranz) [inline css/controls.css]: We don't need this prefix do we?

File: css/controls.css [hunk 7]
```diff
@@ -187,3 +272,83 @@
...
   background-position: -25px -21px;
 }
+
+.controls-right div {
+  background: #cecfcf url(../resources/controls/controls_icons.png) no-repeat;
+  margin-top: 6px;
+  margin-left: 9px;
+  width: 28px;
+  height: 28px;
+  -webkit-border-radius: 3px;
```
Comment (mjschranz) [inline css/controls.css]: Again, we don't need this prefix right?

File: css/controls.css [hunk 8]
```diff
@@ -187,3 +272,83 @@
...
+  cursor: pointer;
+}
+
+
+/* other misc styles to be refactored into embed.less */
+
+
+#post-roll {
+  font-family: "ff-meta-web-pro";
+  text-shadow: 1px 1px 0px rgba(255,255,255,0.4);
```
Comment (mjschranz) [inline css/controls.css]: remove the px for the 0

File: css/controls.css [hunk 9]
```diff
@@ -187,3 +272,83 @@
...
+}
+
+#controls-fullscreen {
+  background-position: 7px -2px;
+}
+
+#controls-logo {
+  width: 45px;
+  height: 42px;
+  background: url(../resources/controls/controls_icons.png) no-repeat -85px 2px;
```
Comment (mjschranz) [inline css/controls.css:304]: Again, no Idea if the strings are required here or not.

File: css/controls.css [hunk 10]
```diff
@@ -160,23 +139,129 @@
...
+  line-height: 1em;
+  padding-top: 14px;
+  padding-right: 15px;
+  padding-left: 15px;
+  color: #58595b;
+}
+
+#controls-currenttime {
+  color: #1eb771;
+
```
Comment (mjschranz) [inline css/controls.css]: don't really need this newline do we?

File: css/controls.css (hunk 2 above)
Comment (k88hudson) [inline css/controls.css:148]: nope

**Pull request merged**